client/js/components/procedure/StatementSegmentsList/StatementSegment.vue

```diff
@@ -114,12 +114,38 @@
               editor-id="recommendationText"
               :procedure-id="procedureId"
               @insert="text => modalProps.handleInsertText(text)" />
-            <dp-recommendation-modal
-              v-if="segment.hasRelationship('tags')"
+            <dp-modal
               ref="recommendationModal"
-              :procedure-id="procedureId"
-              :segment-id="segment.id"
-              @insert-recommendation="text => modalProps.appendText(text)" />
+              class="recommendation-modal"
+              content-classes="u-2-of-3">
+              <div class="flex width-100p">
+                <h3 class="u-mb">
+                  {{ Translator.trans('segment.recommendation.insert.similar') }}
+                </h3>
+                <dp-icon
+                  v-if="isAiTabActive"
+                  class="inline-block"
+                  icon="ai" />
+              </div>
+              <dp-tabs
+                :active-id="activeId"
+                @change="(id) => setActiveTabId(id)">
+                <dp-tab
+                  :label="Translator.trans('segments.recommendations.tags')"
+                  id="recTags">
+                  <addon-wrapper
+                    hook-name="tag.recommendation.tab"
+                    :addon-props="addonProps"/>
+                </dp-tab>
+                <dp-tab
+                  :label="Translator.trans('segments.recommendations.oracle')"
+                  id="recContent">
+                  <addon-wrapper
+                    hook-name="oracle.recommendation.tab"
+                    :addon-props="addonProps"/>
+                </dp-tab>
```

Comment:
we should need ony one addon-wrapper here.

Reply:
The addon wrapper could fetch more than one addon from a single hook but we can not add multiple hooks to a single addon wrapper. Since these addons will be provided by two different addon repositories (and therefore have 2 different hooks) we need the addon wrapper twice

Reply:
and therefor both addons should mount on the same hook. Thats the idea behind it.
we don't need a hook-name for each repos. We should have several dedicated hooks in core where any addon can register to.

Reply:
and the DpTag component would then be the outermost component for each addon?

Reply:
Well I see what was the issue and did a little refactoring here:
https://github.com/demos-europe/demosplan-core/pull/1486/commits/93bc79d6b3b83d1df89db41e1e55a5fe69c96482
https://github.com/demos-europe/demosplan-core/pull/1486/commits/118622924c1cd6775a8000a1e8a5778b914ba333
https://github.com/demos-europe/demosplan-core/pull/1486/commits/467c192120f6c5f6dbaac1508b324cd4aeb31b88
https://github.com/demos-europe/demosplan-core/pull/1486/commits/e84bd07d3be068875f66d2299b28106a869c2134